tox.ini

```diff
@@ -0,0 +1,18 @@
+[tox]
+envlist = py27, py33, py34, py35, py36, pypy3, flake8
+skip_missing_interpreters = True
+
+[testenv]
+deps =
+  -rrequirements.txt
```

Comment:
Nope, this gets passed as-is to pip, so it's `pip -rrequirements` (the same as `pip -r requirements`, but this syntax won't work via tox).